migration/migrating_3_4/deploying-cam.adoc

```diff
@@ -6,7 +6,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Deploying the Cluster Application Migration (CAM) tool requires installing the Cluster Application Migration Operator on the xref:../../migration/migrating_3_4/deploying_cam.adoc#installing-cam-operator-ocp-3_sourcecluster-3-4[{product-title} 3 source] and xref:../../migration/migrating_3_4/deploying_cam.adoc#installing-cam-operator-ocp-4_targetcluster-3-4[{product-title} {product-version} target] clusters.
+You can deploy the Cluster Application Migration (CAM) tool by installing the Cluster Application Migration Operator on the {product-title} 3 source cluster and {product-version} target cluster.
+
+After installing the Cluster Application Migration Operator, you can launch the CAM web console.
 
 :context: sourcecluster-3-4
```

Comment:
This PR is testing the limits of our (docs.openshift.com) implementation of modular docs :). I don't know how this PR is gonna go on the customer portal, now, and in the future implementation of FCC.

A few things: 

* Why is the context being switched here? The context for this assembly should stay at migrating-3-4 as defined earlier in this file.

* Assemblies shouldn't have an id with an undefined context (see the first line) https://github.com/openshift/openshift-docs/pull/22017/files#diff-db29aa1eeb8fc2419fcf94ed09c98bd5R1. This assembly should match the file name: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata. 

* I think you mean to define a new attribute and then unset that attribute, rather than changing the context. You have defined the attribute sourcecluster-3-4 and then unset it immediately after that is used in the assembly, which is ok. But, I also think that instead of creating these new attributes, all you need to do is to leave the context that is already defined (migrating-3-4). The included modules will pick that context up and create the unique ids as required. However, that is not what you really need to do here.. see below.

Overall, as we discussed late last week, this is going to be complex and difficult. My suggestion is:

Create unique assemblies for each migration. So instead of deploy-cam.adoc, create deploy-cam-3-4.adoc, deploy-cam-4-1-4-3.adoc etc, and then include the modules that are common to all these. That will still give you the separate navigation, and you can reuse modules and content.

Reply:
> Why is the context being switched here? The context for this assembly should stay at migrating-3-4 as defined earlier in this file.

The context is being switched because the same module (installing CAM on 4.x cluster) is used for installing on a target or on a source cluster. **However**, there are significant differences between the installations, depending on whether the CAM is installed on the target or source cluster. These differences are tagged, so they require separate contexts. BTW, I removed these links, since I didn't think they were necessary and created additional possible points of failure. :-)

> Assemblies shouldn't have an id with an undefined context 

OK. Will check for undefined context and match assembly to file name.

> Create unique assemblies for each migration. 

Will do. That looks like the safest course of action.